content/cloud/noops-sql-db.md

```diff
@@ -1,29 +1,30 @@
 title = "NoOps SQLite Database"
 template = "cloud_main"
 date = "2023-07-26T16:00:00Z"
-enable_shortcodes = false
+enable_shortcodes = true
 [extra]
 url = "https://github.com/fermyon/developer/blob/main/content/cloud/noops-sql-db.md"
 
 ---
 
 - [Accessing Private Beta](#accessing-private-beta)
 - [Quotas And Service Limitations For the Database in Fermyon Cloud](#quotas-and-service-limitations-for-the-database-in-fermyon-cloud)
-- [How To:](#how-to)
-	- [Prepare a Database](#prepare-a-database)
-	- [Manage the Database Lifecycle](#manage-the-database-lifecycle)
+- [How To](#how-to)
+  - [Database Access From Many Components](#database-access-from-many-components)
+  - [Preparing Fermyon Cloud Schema](#preparing-fermyon-cloud-schema)
+  - [Deleting the Database](#deleting-the-database)
 - [FAQ](#faq)
 - [Next Steps](#next-steps)
 
-With NoOps SQL Database using SQLite support in Fermyon Cloud, you can persist relational data generated by your Spin application in between application invocations without having to worry about database management. Fermyon Cloud will provision and manage the database on your behalf, no operations (NoOps) required. Fermyon Cloud will do the heavy lifting for you. To learn more about the Spin SQLite SDK, please visit the [API guide](../spin/sqlite-api-guide.md).
+With NoOps SQL Database using SQLite support in Fermyon Cloud, you can persist relational data generated by your [Spin](../spin/install.md) application in between application invocations without worrying about database management. Fermyon Cloud will provision and manage the database on your behalf; no operations (NoOps) are required. Fermyon Cloud will do the heavy lifting for you. To learn more about the Spin SQLite SDK, please visit the [API guide](../spin/sqlite-api-guide.md).
```

Comment:
style nit: for me the change to "we do X., no operations required" to "we do X; no operations are required" is less punchy.

Reply:
Thanks, I reverted in next push.